pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/seed-a-registry/index.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: layout.pug
+navigationTitle: Seed Docker Registry
+title: Seed Docker Registry
+menuWeight: 10
+excerpt: Seed your docker registry
+enterprise: false
+---
+
+## Seed your docker registry
+
+Before creating a Kubernetes cluster you must have the required images in a local docker registry. This registry must be accessible from both the bastion machine and the AWS EC2 instances that will be created for the Kubernetes cluster.
+
+1.  Download the images bundle.
+
+    ```bash
+    curl -o konvoy-image-bundle.tar.gz -O downloads.d2iq.com/dkp/v2.2.0/konvoy_image_bundle_v2.2.0_linux_amd64.tar.gz
```

Comment:
```suggestion
    curl -o konvoy-image-bundle.tar.gz -O https://downloads.d2iq.com/dkp/v2.2.0/konvoy_image_bundle_v2.2.0_linux_amd64.tar.gz
```
also, the link currently does not exist. I assume it will be once it is released, right?